packages/odata/queryable.ts

```diff
@@ -13,6 +13,13 @@ import { PipelineMethod } from "./pipeline";
 import { IODataParser, ODataParser } from "./parsers";
 
 export function cloneQueryableData(source: Partial<IQueryableData>): Partial<IQueryableData> {
+  let body;
+  // this handles bodies that cannot be JSON encoded (Blob, etc)
+  // Note however, even bodies that can be serialized will not be cloned.
```

Comment:
The original code would not attempt to clone the body either.
I didn't change that behavior. 

It was doing all the work to `JSON.stringify` it though and that's where it failed if passed Node's IncomingMessage type stream with all the response and network additions.

Reply:
This should address https://github.com/pnp/pnpjs/issues/1261